### PR TITLE
feat: project details ui

### PR DESCRIPTION
**Pull request by mo-cherif**
## Description
This pr is about the poject details interface.

part of #137 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: frontend/src/components/service-cards.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        <p className="flex gap-2 items-center">{updatedAt}</p>
+      </CardContent>
+
+      <>
+        <Separator />
+        <CardFooter className="p-0 text-gray-400 px-6 py-4 text-sm flex gap-2">
+          <HardDrive size={20} /> {volumeNumber}
+          {volumeNumber > 1 ? " Volumes" : " Volume"}
+        </CardFooter>
+      </>
```
Comment (Fredkiss3) [inline frontend/src/components/service-cards.tsx:125]:
```suggestion
      <Separator />
      <CardFooter className="p-0 text-gray-400 px-6 py-4 text-sm flex gap-2">
        <HardDrive size={20} /> {volumeNumber}
        {volumeNumber > 1 ? " Volumes" : " Volume"}
      </CardFooter>
```

File: frontend/src/index.css [hunk 2]
```diff
@@ -75,6 +76,7 @@
       --toggle: 164 43% 2%;
       --ring: 164 61% 70%;
       --loader: white;
+      --link:#60a5fa;
```
Comment (Fredkiss3) [inline frontend/src/index.css:79]:
```suggestion
      --link: #60a5fa;
```

File: frontend/src/components/service-cards.tsx [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+              ></span>
+            </span>
+          </TooltipTrigger>
+          <TooltipContent>
+            <div>
+              {status === "healthy" && "✅ "}
+              {status === "sleeping" && "💤 "}
+              {status === "unhealthy" && "❌ "}
+              {status === "undeployed" && "⏳ "}
+              {status}
```
Comment (Fredkiss3) [inline frontend/src/components/service-cards.tsx:82]: The text should be `not deployed yet` if the status is `undeployed`

File: frontend/src/components/service-cards.tsx [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
```
Comment (Fredkiss3) [inline frontend/src/components/service-cards.tsx:189]: The text should be `not deployed yet` if the status is `undeployed`

**Review by Fredkiss3 — CHANGES_REQUESTED** [review]
Overall everything is good, some small changes are needed though.

File: frontend/src/components/service-cards.tsx (hunk 3 above)
Comment (mo-cherif) [reply frontend/src/components/service-cards.tsx:82]: didn't understand what you mean

File: frontend/src/components/service-cards.tsx [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+              ></span>
+            </span>
+          </TooltipTrigger>
+          <TooltipContent>
+            <div>
+              {status === "healthy" && "✅ "}
+              {status === "sleeping" && "💤 "}
+              {status === "unhealthy" && "❌ "}
+              {status === "undeployed" && "Not deployed yet ⏳ "}
+              {status === "undeployed" ? "" : status}
```
Comment (Fredkiss3) [inline frontend/src/components/service-cards.tsx:82]:
```suggestion
              {status === "undeployed" && "⏳ "}
              {status === "undeployed" ? "Not deployed yet" : status}
```

File: frontend/src/components/service-cards.tsx [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+              ></span>
+            </span>
+          </TooltipTrigger>
+          <TooltipContent>
+            <div>
+              {status === "healthy" && "✅ "}
+              {status === "sleeping" && "💤 "}
+              {status === "unhealthy" && "❌ "}
+              {status === "undeployed" && "⏳ "}
+              {status}
```
Comment (Fredkiss3) [inline frontend/src/components/service-cards.tsx:187]:
```suggestion
              {status === "undeployed" ? "Not deployed yet" : status}
```

**Review by Fredkiss3 — APPROVED** [review]
Lgtm

**Pull request merged**